src/pykeen/models/base.py

```diff
@@ -299,6 +300,39 @@ def predict_scores_all_tails(
             scores = torch.sigmoid(scores)
         return scores
 
+    def _get_novelty_mask(
```

Comment:
can we actually make this a static function? I realized it might also be useful to add an additional column to the prediction dataframe to show if a given triple appeared in either the testing/validation set. Since these triples factories aren't stored in the model itself, then this logic will have to be applied by an external function to the model itself

Reply:
Sure.

There might also be overlap with the LCWA preprocessing code (where we group triples that share the same `(h,r)` / `(r,t)` part).

Reply:
Done in 54c720a

Reply:
Do you want to look into combining the code in the scope of this PR? Or are we done?

Reply:
Let's move it to an issue :wink: 

I suppose @lvermue might help us here, since he looked deeply into fast grouping :slightly_smiling_face:

Reply:
I'll address this in #61 .